### PR TITLE
Pin transitive dependencies (#1738)

### PR DESCRIPTION
**Pull request by achave11**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -24,6 +24,40 @@ rsa==3.4.2
 urllib3==1.24.3
 more-itertools==4.3.0
 typing-extensions==3.7.4
-werkzeug==0.15.5
+Werkzeug==0.15.5
```
Comment (jessebrennan) [inline requirements.txt:27]: Why the case change?

File: requirements.dev.txt [hunk 2]
```diff
@@ -5,15 +5,57 @@ locustio==0.9.0
...
 pytest==3.2.3
 coverage==5.1
-gitpython==2.1.11
-gitdb2==3.0.1  # Not a direct dependency but https://github.com/gitpython-developers/GitPython/issues/983
+GitPython==2.1.11
 google-api-python-client==1.7.8
 google-auth==1.14.1
 moto==1.3.13  # 1.3.14 does not work, possibly related to https://github.com/spulec/moto/issues/2567
-pyyaml==5.1
+PyYAML==5.1
```
Comment (jessebrennan) [inline requirements.dev.txt:12]: Why the case change?

File: requirements.dev.txt [hunk 3]
```diff
@@ -5,15 +5,57 @@ locustio==0.9.0
 gevent==20.5.0
 pytest==3.2.3
 coverage==5.1
-gitpython==2.1.11
-gitdb2==3.0.1  # Not a direct dependency but https://github.com/gitpython-developers/GitPython/issues/983
+GitPython==2.1.11
```
Comment (jessebrennan) [inline requirements.dev.txt:8]: Why the case change?

File: requirements.txt (hunk 1 above)
Comment (achave11) [reply requirements.txt:27]: Note the name on the url and the package name on PyPI
https://pypi.org/project/werkzeug/

File: requirements.dev.txt (hunk 2 above)
Comment (achave11) [reply requirements.dev.txt:12]: https://pypi.org/project/pyyaml/

File: requirements.dev.txt (hunk 3 above)
Comment (achave11) [reply requirements.dev.txt:8]: https://pypi.org/project/gitpython/

**Comment by achave11** [comment]
> It would be nice if there was a convenient way to update dependencies... I suppose that should be a different issue though.
> 
> I want to be sure the change of the case is not a mistake before I approve.

Note that I introduced a new commit with the renaming. I believe they should be listed as they are displayed in the PyPI library.

**Comment by hannes-ucsc** [comment]
Superseded by #1882

**Pull request closed without merging**